src/scripts/quick_reblog.js

```diff
@@ -56,6 +59,14 @@ const rememberedBlogStorageKey = 'quick_reblog.rememberedBlogs';
 const quickTagsStorageKey = 'quick_tags.preferences.tagBundles';
 const blogHashes = {};
 
+const renderBlogAvatar = async () => {
+  const { value: selectedUuid } = blogSelector;
+  const { avatar } = userBlogs.find(({ uuid }) => uuid === selectedUuid);
+  const { url } = avatar[avatar.length - 1];
```

Comment:
```suggestion
  const { url } = avatar.at(-1);
```

Reply:
I got super excited about this when I discovered it recently before I realized that it would require a minimum version bump. Do we care about that? https://caniuse.com/mdn-javascript_builtins_array_at

Reply:
I misremembered our minimum versions! Probably not worth doing, then.

I guess a different way to shortcut it would be
```suggestion
  const [{ url }] = avatar.slice(-1);
```
but it's too late at night for me to tell if this is silly or not.

Reply:
To be fair, it's Firefox 89 -> 90 and Chromium 89 -> 92, so a very small bump. Requiring 2022 Safari if and when Safari support becomes a thing at all is the more notable change browser-release-date-wise, but you know... shrug.